lib/pharos/addon.rb

```diff
@@ -1,6 +1,14 @@
 # frozen_string_literal: true
 
 require 'dry-validation'
+require 'fugit'
+
+module Fugit
+  class Cron
+    attr_reader :seconds # XXX: missing from upstream
```

Comment:
We should try to push this also to the upstream.

Reply:
https://github.com/floraison/fugit/pull/3

Reply:
^ this got merged and released upstream as 1.1.2, so I'll still update this PR to use that instead.